### PR TITLE
Add titles to generated compare models graphs

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Design Change:
- When a user compares multiple models generating a graph. There were no titles added to the image generated.

Code Changes:
- added a KernelManager message to to request the title pair(s) of the compared models 

image

Resolves #(issue)

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue [hunk 1]
```diff
@@ -248,6 +262,17 @@ function runCode() {
 	emptyImages();
 	updateCodeState();
 
+	kernelManager.sendMessage('get_comparison_pairs_request', {}).register('any_get_comparison_pairs_reply', (data) => {
+		const comparisonPairs = data.msg.content?.return?.comparison_pairs;
+		const state = cloneDeep(props.node.state);
```
Comment (Tom-Szendrey) [inline packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue:267]: No need to clone state here

**Review by Tom-Szendrey — APPROVED** [review]
Sweet

File: packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue [hunk 2]
```diff
@@ -198,6 +198,7 @@ const code = ref(props.node.state.notebookHistory?.[0]?.code ?? '');
 const llmThoughts = ref<any[]>([]);
 const isKernelReady = ref(false);
 const contextLanguage = ref<string>('python3');
+const compareModelTitles = ref(props.node.state.comparisonPairs);
```
[the remark this answers is not in the document]
Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue:201]: making change

File: packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue (hunk 1 above)
Comment (asylves1) [reply packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue:267]: It's for the `else if`  when comparisonPairs returns nothing. I only update the titles using the past state if something has been stored previously

**Pull request merged**